### PR TITLE
CI: bump FreeBSD to 13.2 and 14.0

### PR DESCRIPTION
**Pull request by clementperon**
Test FreeBSD on Cirrus

### CODE REVIEW

File: Pcap++/src/PcapLiveDevice.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@
 // On Mac OS X and FreeBSD timeout of -1 causes pcap_open_live to fail so value of 1ms is set here.
 // On Linux and Windows this is not the case so we keep the -1 value
 #if defined(__APPLE__) || defined(__FreeBSD__)
-#define LIBPCAP_OPEN_LIVE_TIMEOUT 1
+#define LIBPCAP_OPEN_LIVE_TIMEOUT 10
```
Comment (seladb) [inline Pcap++/src/PcapLiveDevice.cpp:45]: I wonder if it works with a lower value like 2ms 🤔 
Also - maybe separate FreeBSD and Apple? Apple could stay 1ms and we can have a different value for FreeBSD

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:45]: I will do the test, but we need to found the proper fix.
This is just to highlight that it's not an issue with the CI or the user privilege that has changed.

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:45]: maybe we should move to a nonblock version ?

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:45]: Timeout 1ms: 3x KO
Timeout 2ms: 3x KO
Timeout 3ms: 3x KO
Timeout 4ms: 3x KO
Timeout 5ms: 3x KO
Timeout 6ms: 3x KO
Timeout 7ms: 3x KO
Timeout 9ms: 3x KO
Timeout 10ms: 3x OKKK
Timeout 11ms: 3x OKKK
Timeout 15ms: 3x OKKK
Timeout 20ms: 3x OKKK
Timeout 30ms: 3x OKKK

Comment (seladb) [reply Pcap++/src/PcapLiveDevice.cpp:45]: @clementperon I'm not sure what this output means 🤔

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:45]: Sorry not clear, I test each configuration 3 times to be sure it was repeatable

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:45]: To be honest I don't know, I have tested that on a FreeBSD VM Aarch64 on MacOS.

I'm not sure we should merge this without understand why no?

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:49]: I have pushed an update with a proper comment

**Review by seladb — APPROVED** [review]

File: Pcap++/src/PcapLiveDevice.cpp (hunk 1 above)
Comment (seladb) [reply Pcap++/src/PcapLiveDevice.cpp:49]: > To be honest I don't know, I have tested that on a FreeBSD VM Aarch64 on MacOS.
> 
> I'm not sure we should merge this without understand why no?

Honestly, I'm not sure how we can test it... seems like an internal issue of libpcap on FreeBSD 🤷‍♂️

Comment (clementperon) [reply Pcap++/src/PcapLiveDevice.cpp:49]: Ideally we would open a bug with  a small sample code to reproduce the issue

Comment (seladb) [reply Pcap++/src/PcapLiveDevice.cpp:49]: Sure, you can open a bug. But even if they fix it we need to support older libpcap versions, so we'll need this hack anyway...

**Comment by tigercosmos** [comment]
thanks for the PR. waiting for this merged.

**Comment by seladb** [comment]
@clementperon I'm merging this PR because is blocks other PRs. We can try to figure out the issue with FreeBSD 14 later and maybe fix it

**Pull request merged**